modules/nested-tabs/assets/js/frontend/handlers/nested-tabs.js

```diff
@@ -305,7 +309,37 @@ export default class NestedTabs extends Base {
 		}
 
 		if ( ! isActiveTab ) {
+			const isMobileVersion = 'none' === this.elements.$headingContainer.css( 'display' );
+
+			if ( isMobileVersion ) {
+				this.activateMobileTab( tabIndex );
+				return;
+			}
+
+			this.activateTab( tabIndex );
+		}
+	}
+
+	activateMobileTab( tabIndex ) {
+		// Timeout added to ensure that opening of the active tab starts after closing the other tab.
+		setTimeout( () => {
 			this.activateTab( tabIndex );
+			this.forceActiveTabToBeInViewport( tabIndex );
+		} );
+	}
+
+	forceActiveTabToBeInViewport( tabIndex ) {
+		if ( ! elementorFrontend.isEditMode() ) {
+			return;
+		}
+
+		const activeClass = this.getActiveClass(),
+			$activeTabTitle = this.elements.$mobileTabTitles.filter( this.getTabTitleFilterSelector( tabIndex ) ),
+			boundingBox = $activeTabTitle[ 0 ].getBoundingClientRect(),
+			isActiveTabTitleInViewport = boundingBox.top >= 0 && boundingBox.bottom <= window.innerHeight;
```

Comment:
@hein-obox Can you use `elementor.helpers.isInViewport( element, html )` to determine if the element is in the viewport maybe?

Reply:
@louiswol94 That is a really valuable suggestion. Done.